edb/lib/std/30-strfuncs.eql

```diff
@@ -19,11 +19,84 @@
 ## String functions
 
 
-# std::lower
-# ----------
+CREATE FUNCTION
+std::ascii(val: std::str) -> std::int64
+{
+    FROM SQL FUNCTION 'ascii';
+};
+
 
 CREATE FUNCTION
-std::lower(str: std::str) -> std::str
+std::chr(val: std::int64) -> std::str
+{
+    FROM SQL $$
+    SELECT chr(val::int4)
+    $$;
+};
+
+
+CREATE FUNCTION
+std::lower(s: std::str) -> std::str
 {
     FROM SQL FUNCTION 'lower';
 };
+
+
+CREATE FUNCTION
+std::upper(s: std::str) -> std::str
+{
+    FROM SQL FUNCTION 'upper';
+};
+
+
+CREATE FUNCTION
+std::title(s: std::str) -> std::str
```

Comment:
I think spelling this out as `titlecase()` would be clearer. Likewise for `lower()` and `upper()`.

Reply:
I think they all should have "str_" prefix. I'm -0 on the "case" suffix.

Reply:
I've added the "str_" prefix.